js/apps/admin-ui/src/user/EditUser.tsx

```diff
@@ -152,7 +152,7 @@ const EditUserForm = ({ user, bruteForced, refresh }: EditUserFormProps) => {
     const attributes =
       "key" in (formUser.attributes?.[0] || [])
         ? keyValueToArray(formUser.attributes as KeyValueType[])
-        : user.attributes;
```

Comment:
I feel like this has been added with some intent to fix a bug, but I could be wrong. Just to be sure, have you tried the following:

- Add a couple of attributes to a user and save.
- Reload the page and change some other field, leaving the attributes, then save the user.
- Reload the page again and save the attributes.

I feel like perhaps if the attributes are left untouched their value in the form might be left empty, hence the fallback to the initial value of the user attributes.

Reply:
Perfect, thanks for verifying this!